### ISSUE TITLE
false positive on SimpleXmlElement passing..

### ISSUE THREAD

**Issue by ppetermann** [scrutinizer-ci/php-analyzer#465]
> Here, you can explain why the found issue was invalid, and what result
> you would have expected otherwise.
>
> Please also check that this issue has not already been reported.
uhm, $response->xml() returns a \SimpleXmlElement, which has a child named params, thus this works, not sure where the claim it doesn't come from.

see https://scrutinizer-ci.com/g/ppetermann/devedge-xmlrpc-client/inspections/8908a3a4-727b-470d-8d9b-f13940f48e53/issues/files/src/Devedge/XmlRpc/Client.php?status=new&orderField=path&order=asc&issueId=f44fa4f8-7ab6-449e-9a3e-af704dbd276b

**Comment by schmittjoh** [comment]
This is a valid issue unfortunately. The problem here is not the the property ``params``, but passing a function call to a function that expects a reference. 

When running with strict standards, PHP will raise an issue such as this:

```
PHP Strict Standards:  Only variables should be passed by reference in abc.php on line 123
```
